src/Texts/Texts.js

```diff
@@ -0,0 +1,28 @@
+const texts = {
+  componentsTitles: {
+    character: 'Character Info',
+    coinsButton: 'Coins',
+    energyButton: 'Energy',
+    disaster: 'Disaster',
+    zoomSlider: 'ZoomSlider',
+    mapCompass: 'MapCompass',
+    actions: 'Actions',
+    seen: 'Seen',
+    crafting: 'Crafting',
+  },
+  buttonsText: {
+    getMore: 'Get More',
+  },
+  coins: {
+    amountPt1: 'You have ',
+    amountPt2: 'Coins.',
+  },
+  energy: {
+    energySymbol: '♠',
+    fullIn: 'Full Energy in:',
+    rate: 'Rate: ',
+    energyPerHour: '♠/h',
+  },
+};
+export default texts;
+export const { componentsTitles, buttonsText, energy, coins } = texts;
```

Comment:
should put more thought into the hirarchi of this element and how to export it.
maybe it should be built around fields of context, such as ui vs content, etc.
maybe by compontns in code?
the goal is that it'll be more intuitive to be imported and used in the components.